clone-all

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+username='jenkinsci'
+if [ -z "$1" ]; then
+  echo "Using jenkinsci/* for clones";
+else
+  echo "Using '$1' for clones";
+  username=$1
+fi
+
+git clone git@github.com:$username/workflow-step-api-plugin.git ../workflow-step-api-plugin
```

Comment:
More maintainable use Perl (or Ruby, or Bash + `sed`, or whatever) to grep `demo/plugins.txt` (and, at least until #1 is merged, `demo/snapshot-only-plugins.txt`) for `workflow-*` or `pipeline-*` plugins.

The exception is `pipeline-stage-view`, which together with `pipeline-rest-api` lives in `pipeline-stage-view-plugin`. Not currently a dependency here, but probably will be.

Also you forget `workflow-multibranch`, it seems.